library/blivet.py

```diff
@@ -559,6 +590,40 @@ def _create(self):
             label = get_format("disklabel", device=self._device.path, label_type=disklabel_type)
             self._blivet.format_device(self._device, label)
 
+class BlivetMDRaidPool(BlivetPool):
```

Comment:
I don't think this should be a subclass of `BlivetPool`, but I do think it might make sense to have some shared code for creating the arrays. The same will be true for encryption.

Reply:
BlivetMDRaidPool is no longer a subclass of BlivetPool.